closure/templates/test/BUILD

```diff
@@ -58,6 +58,38 @@ closure_js_template_library(
     ],
 )
 
+closure_js_template_library(
+    name = "plugin_soy",
+    srcs = ["plugin.soy"],
+    plugin_modules = ["test.ExampleModule"],
+    plugin_module_deps = [":ExampleModule"],
+)
+
+java_library(
+    name = "ExampleModule",
+    srcs = ["ExampleModule.java"],
+    deps = [
+        "@com_google_guava//:com_google_guava",
```

Comment:
Nit: Maybe use `@com_google_guava//jar`?

Reply:
Ah, I blindly copied/pasted the buildozer "Pro-tip". Updated